ydb/docs/en/core/integrations/liquibase.md

```diff
@@ -0,0 +1,314 @@
+                <constraints primaryKey="true"/>
+            </column>
+            <column name="season_id" type="bigint">
+                <constraints primaryKey="true"/>
+            </column>
+            <column name="episode_id" type="bigint">
+                <constraints primaryKey="true"/>
+            </column>
+
+            <column name="title" type="text"/>
+            <column name="air_date" type="timestamp"/>
+        </createTable>
+        <rollback>
+            <dropTable tableName="episodes"/>
+        </rollback>
+    </changeSet>
+    <changeSet id="index_episodes_title" author="kurdyukov-kir">
+        <createIndex tableName="episodes" indexName="index_episodes_title" unique="false">
+            <column name="title"/>
+        </createIndex>
+    </changeSet>
+</databaseChangeLog>
+```
+
+After executing the `liquibase update` command, Liquibase will print the following log.
+
+```bash
+UPDATE SUMMARY
+Run:                          2
+Previously run:               0
+Filtered out:                 0
+-------------------------------
+Total change sets:            2
+
+Liquibase: Update has been successful. Rows affected: 2
+Liquibase command 'update' was executed successfully.
+```
+
+After applying migrations, the data schema now looks like this:
+
+![../_assets/liquibase-step-1.png](../_assets/liquibase-step-1.png)
+
+You can see that Liquibase has created two service tables: `DATABASECHANGELOG`, which is the migration log, and `DATABASECHANGELOGLOCK`, which is a table for acquiring a distributed lock.
+
+Contents of the `DATABASECHANGELOG` table:
+
+| AUTHOR        | COMMENTS        | CONTEXTS | DATEEXECUTED | DEPLOYMENT_ID | DESCRIPTION                                                    | EXECTYPE | FILENAME               | ID                   | LABELS | LIQUIBASE | MD5SUM                             | ORDEREXECUTED | TAG |
+|:--------------|:----------------|:---------|:-------------|:--------------|:---------------------------------------------------------------|:---------|:-----------------------|:---------------------|:-------|:----------|:-----------------------------------|:--------------|:----|
+| kurdyukov-kir | Table episodes. |          | 12:53:27     | 1544007500    | createTable tableName=episodes                                 | EXECUTED | migration/episodes.xml | episodes             |        | 4.25.1    | 9:4067056a5ab61db09b379a93625870ca | 1             |
+| kurdyukov-kir | ""              |          | 12:53:28     | 1544007500    | createIndex indexName=index_episodes_title, tableName=episodes | EXECUTED | migration/episodes.xml | index_episodes_title |        | 4.25.1    | 9:49b8b0b22d18c7fd90a3d6b2c561455d | 2             |
+
+### Evolution of the database schema
+
+Let's say we need to create a {{ydb-short-name }} topic and turn off auto partitioning of the table. This can be done with a native SQL script:
+
+```sql
+--liquibase formatted sql
+
+--changeset kurdyukov-kir:10
```

Comment:
It's not clear if this number has any meaning. Also, why does it use your login for the name instead of something like `create-a-topic`?

Reply:
kurdyukov-kir is author, 10 is id